script.skin.helper.skinbackup/service.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+'''
+    script.skin.helper.skinbackup
+    Kodi addon to backup skin settings
+'''
+
+from resources.lib.colorthemes import ColorThemes
+from resources.lib.backuprestore import BackupRestore
+from resources.lib.utils import log_msg
+import xbmc
+import _strptime
+
+
+class Service():
+    '''Background service for automatic skin backups and/or automatic colortheme switches'''
+
```

Comment:
OK, checked the code. Can never happen because it's controlled by skin settings so if you switch to another skin, that skin will have other settings. It's fail safe.